common/utils/src/main/resources/error/error-classes.json

```
@@ -2446,6 +2446,11 @@
     ],
     "sqlState" : "42P01"
   },
+  "TABLE_VALUED_FUNCTION_FAILED_TO_ANALYZE_IN_PYTHON" : {
+    "message" : [
+      "The user defined Python table function failed to analyze in Python, <msg>"
```

```suggestion
      "Failed to analyze the Python user defined table function: <msg>"
```
Is it possible to add the function name?